api/v1alpha1/systemconfiguration_types.go

```diff
@@ -42,7 +49,8 @@ type SystemConfigurationComputeNodeReference struct {
 // SystemConfigurationStorageNode describes a storage node in the system
 type SystemConfigurationStorageNode struct {
 	// Type is the type of server
-	// +kubebuilder:validation:Enum=Rabbit
+	// +kubebuilder:validation:Enum:=Rabbit
+	// +kubebuilder:validation:Type:=string
```

Comment:
make manifests

Reply:
Since it's already a string, there is no change to the CRD when running `make manifests`. I only include the `Type:=string` so it is obvious to the observer that StorageNodeType is a string. (I did the same for WorkflowState)